tests/mcs_benchmark_data_test/pipelines/social_iqa/social_iqa_pipeline_test.py

```diff
@@ -0,0 +1,36 @@
+import bz2
+from io import StringIO
+
+from rdflib import Graph
+
+from mcs_benchmark_data.path import DATA_DIR_PATH
+from mcs_benchmark_data.pipelines.social_iqa.social_iqa_benchmark_pipeline import (
+    SocialIQaBenchmarkPipeline,
+)
+from mcs_benchmark_data.pipelines.social_iqa.social_iqa_benchmark_file_names import (
+    SocialIQaBenchmarkFileNames,
+)
+
+
+def test_extract_transform_load():
+    SocialIQaBenchmarkPipeline(
+        file_names=SocialIQaBenchmarkFileNames(
+            metadata="metadata.json",
+            dev_labels="dev-labels.lst",
+            dev_samples="dev.jsonl",
+            train_labels="train-labels.lst",
+            train_samples="train.jsonl",
+        ),
+    ).extract_transform_load()
+
+    loaded_data_dir_path = DATA_DIR_PATH / "loaded" / SocialIQaBenchmarkPipeline.ID
```

Comment:
Feel free to move this assertion code into a reusable function, can just drop it in a file somewhere in the tests/ directory. Like

```
def assert_valid_rdf_loaded(pipeline_id: str):
...
```